src/validation_workflow/api_test_cases.py

```diff
@@ -0,0 +1,46 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+import time
+from typing import Any
+
+from validation_workflow.api_request import ApiTest
+from validation_workflow.validation_args import ValidationArgs
+
+'''
+This class is the collection of test cases to run.
+'''
+
+
+class TestCases:
```

Comment:
Maybe name this as ApiTestCases? Its more clear.

Reply:
Thanks, @gaiksaya , sounds good, let us update it at next commit.